src/bootstrap/util.rs

```diff
@@ -607,3 +607,25 @@ pub fn get_clang_cl_resource_dir(clang_cl_path: &str) -> PathBuf {
     let clang_rt_dir = clang_rt_builtins.parent().expect("The clang lib folder should exist");
     clang_rt_dir.to_path_buf()
 }
+
+struct LldNoThreads {
+    windows: &'static str,
+    other: &'static str,
+}
+
+lazy_static::lazy_static! {
```

Comment:
Already depends on `once_cell`, can't this be implemented with it instead of `lazy_static`?

Reply:
You are right, I suggested using `lazy_static` because `once_cell` from the standard library is unstable and cannot be used in bootstrap. But if it can be used from a third-party crate, then it would be better to use it instead of `lazy_static`.